CONTRIBUTING.md

```diff
@@ -86,20 +86,20 @@ source ~/.bash_profile
 Accept the SDK licences:
 
 ```
-$ANDROID_SDK_ROOT/tools/bin/sdkmanager --licenses
+$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --licenses
```

Comment:
Does this mean the paths changed in a newer version of the SDK?

In which case would there be a minimum version of supported SDK we need to mention, as my path is still the old one and it works

Reply:
I completely forgot about this PR 😂 it had to do with the switch to Java 11 - I don't remember the details of what went wrong and where but there were issues with the old sdkmanagers with Java 11